# HG changeset patch
# User John Donoghue
# Date 1530814633 14400
#      Thu Jul 05 14:17:13 2018 -0400
# Branch stable
# Node ID 65a28dec405b77afd828fee864ffa7a5c0788a11
# Parent  9277b77dd28fbb494d35ea18f1904a747eeffdfa
use octave bat file in pkg install configure/make when in windows (Bug #53938)

* scripts/pkg/private/configure_make.m:
  set octave_binary to octave-version.bat if in windows

diff --git a/scripts/pkg/private/configure_make.m b/scripts/pkg/private/configure_make.m
--- a/scripts/pkg/private/configure_make.m
+++ b/scripts/pkg/private/configure_make.m
@@ -29,17 +29,21 @@ function configure_make (desc, packdir, 
     src = fullfile (packdir, "src");
     octave_bindir = __octave_config_info__ ("bindir");
     ver = version ();
     ext = __octave_config_info__ ("EXEEXT");
     mkoctfile_program = fullfile (octave_bindir, ...
                                   sprintf ("mkoctfile-%s%s", ver, ext));
     octave_config_program = fullfile (octave_bindir, ...
                                       sprintf ("octave-config-%s%s", ver, ext));
-    octave_binary = fullfile (octave_bindir, sprintf ("octave-%s%s", ver, ext));
+    if (ispc () && ! isunix ())
+      octave_binary = fullfile (octave_bindir, sprintf ("octave-%s.bat", ver));
+    else
+      octave_binary = fullfile (octave_bindir, sprintf ("octave-%s%s", ver, ext));
+    endif
 
     if (! exist (mkoctfile_program, "file"))
       __gripe_missing_component__ ("pkg", "mkoctfile");
     endif
     if (! exist (octave_config_program, "file"))
       __gripe_missing_component__ ("pkg", "octave-config");
     endif
     if (! exist (octave_binary, "file"))
